### ISSUE TITLE
Not working in either normal download or through pie

### ISSUE THREAD

**Issue by coreyplasse** [chicueloarcade/es-theme-Chicuelo#31]
theme is no longer working and shows nothing when downloaded

**Comment by RedGreen007** [comment]
Same. The git seems to be broken and all the files removed. I installed this theme over a year ago on a friends Pi and still have the files there. I dont know if he just removed them or what.

**Comment by ansostuhmieng** [comment]
I agree, not working when downloaded via emulationstation/retropie tools.  I was able to get it working by manually rolling back to this commit: https://github.com/chicueloarcade/es-theme-Chicuelo/commit/df5c015e14b971df4c60f781d4986791647943d1
